controllers/k8ssandra/k8ssandracluster_controller_test.go

```diff
@@ -833,7 +833,7 @@ func createMultiDcCluster(t *testing.T, ctx context.Context, f *framework.Framew
 		}
 
 		condition := FindDatacenterCondition(k8ssandraStatus.Cassandra, cassdcapi.DatacenterScalingUp)
-		return condition != nil && condition.Status == corev1.ConditionTrue
+		return !(condition == nil && condition.Status == corev1.ConditionFalse)
```

Comment:
Note with this check that we don't actually care if the condition is set to true. Is that intentional? Why are we checking at all if the state is irrelevant?